test/index/vocabulary/VocabularyInMemoryTest.cpp

```diff
@@ -17,7 +17,8 @@ auto createVocabulary(const std::vector<std::string>& words) {
   auto filename = "vocabInMemoryCreation.tmp";
   {
     Vocab v;
-    auto writer = v.makeDiskWriter(filename);
+    auto writerPtr = v.makeDiskWriterPtr(filename);
+    auto& writer = *writerPtr;
     for (const auto& [i, word] : ::ranges::views::enumerate(words)) {
```

Comment:
Unrelated to this PR but I've seen it twice now. Does `::ranges` refer to the correct `ql::ranges` namespace for C++17 compatibility?

Reply:
No, 
`::ranges` should actually be called `range-v3::` but isn't.
We can't use `ql::ranges` because our `std::ranges` don't have enumerate yet...

Reply:
Makes sense